src/main/resources/en.yml

```diff
@@ -241,3 +241,9 @@ en:
+       measurements: ["teaspoon", "tablespoon", "cup", "pint", "quart", "gallon"]
```

Comment:
I guess this could be much improved, but this pull request is just a simple copy of https://github.com/stympy/faker/commit/f3e08e67e8964017b999da09d09375b3befbd0d1